src/lang-go.ts

```diff
@@ -1,6 +1,8 @@
 import '@babel/polyfill'
 
 import { Handler, initLSIF, asyncFirst, wrapMaybe, Maybe } from '@sourcegraph/basic-code-intel'
+// tslint:disable-next-line: no-submodule-imports
```

Comment:
The warning of this rule is legit and shouldn't be disabled. I added some guidance to https://github.com/sourcegraph/eslint-config#when-to-use-eslint-disable

```suggestion
```

Reply:
lsp-client needs to export the conversion functions from the top level then, right? Would you be interested in doing that? I can get to it too

Reply:
Yes, but it is just warning, so this doesn't break CI